docs/source/tutorials/configuration-and-setup.rst

```diff
@@ -0,0 +1,113 @@
+==============================
+Configuration and plugin setup
+==============================
+
+Maybe you :doc:`played with commands </tutorials/playing-command>` for your
```

Comment:
Broken link. That file was renamed.

```suggestion
Maybe you :doc:`played with commands </tutorials/playing-with-commands>` for your
```

The most recent deploy preview log [flags this](https://app.netlify.com/sites/sopel-unstable-docs/deploys/64e926fbcf9163000742ac0a#L556), but it's hard to find in the sea of other warnings that I believe #2496 will at least reduce:

```
5:12:28 PM: /opt/build/repo/docs/source/tutorials/configuration-and-setup.rst:5: WARNING: unknown document: /tutorials/playing-command
```